templates/etc/rsyslog.conf.j2

```diff
@@ -1,4 +1,4 @@
-# {{ ansible_managed }}
+# {{ ansible_managed | comment }}
```

Comment:
Should this be

```
{{ ansible_managed | comment }}
```

without the leading `# `?

Reply:
yes, `# ` not required.
https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#adding-comments-to-files